.github/workflows/R-CMD-check.yaml

```diff
@@ -0,0 +1,18 @@
+on: [push, pull_request]
+
+name: R-CMD-check
+
+jobs:
+  R-CMD-check:
+    runs-on: macOS-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: r-lib/actions/setup-r@master
```

Comment:
I think if you want use a specific R version, it would go here.

```
      - uses: r-lib/actions/setup-r@master
        with:
          r-version: '3.6.3'
```